lib/swaps/SwapRecovery.ts

```diff
@@ -16,7 +16,7 @@ interface SwapRecovery {
  * ensuring that we do not lose funds on a partially completed swap.
  */
 class SwapRecovery extends EventEmitter {
-  public static readonly PENDING_SWAP_RECHECK_INTERVAL = 300000;
+  public static readonly PENDING_SWAP_RECHECK_INTERVAL = 15000;
```

Comment:
What is the reason for changing this constant?

Reply:
Sorry, forgot to change this back as I was testing the failed swap flows. Changed.